boards/nxp/mimxrt1050_evk/board.cmake

```
@@ -3,10 +3,17 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
+
+if (NOT ("${BOARD_QUALIFIERS}" MATCHES "qspi"
```

```suggestion
if(NOT ("${BOARD_QUALIFIERS}" MATCHES "qspi"
```
No spaces between if and bracket in cmake

---

okay, I'll change this, but just so you know, there are many many board.cmake in tree that don't put a space after the `if`, I don't know if it warrants a mass cleanup

---

what about a space between the `else` and the parentheses? and the `endif`